payment-element/.env.exmaple

```diff
@@ -0,0 +1,5 @@
+STRIPE_PUBLISHABLE_KEY=<replace-with-your-publishable-key>
```

Comment:
typo in the name of this file, should be `.env.example`

Reply:
Good catch